driver-core/src/main/java/com/datastax/driver/core/SchemaParser.java

```diff
@@ -1589,6 +1643,21 @@ SystemRows fetchSystemRows(
                 protocolVersion);
       }
 
+      if (isSchemaOrKeyspace) {
+        if (targetType == KEYSPACE) {
+          scyllaKsFuture =
+              queryAsync(
+                  SELECT_SCYLLA_KEYSPACES
+                      + " WHERE keyspace_name = '"
+                      + targetKeyspace
+                      + "' LIMIT 1;",
+                  connection,
+                  protocolVersion);
+        } else {
+          scyllaKsFuture = queryAsync(SELECT_SCYLLA_KEYSPACES, connection, protocolVersion);
+        }
```

Comment:
`scylla_keyspaces` is not populated with keyspaces created with default configuration, so it is going to fail.

Reply:
In that case it's going to return 0 rows, which is not breaking unless other code explicitly calls for a row that does not exist. 
I'm not sure if I understand the meaning of default here. Scylla 6.2.0 has tablets enabled by default and keyspaces created without explicit CQL syntax to enable tablets do appear in `scylla_keyspaces`. The keyspaces that do not appear are treated as not tablets-based.